### DISCUSSION TITLE
Subscribing partially works

### DISCUSSION THREAD

**Discussion by ghost** [community/community#61173] [Mobile] [answered]
I can unsubscribe from any thread (Issues, PRs, Discussions) however can't subscribe to them.

**Comment by esahmed21** [comment]
Hi @rywz 👋

Thank you for reaching out to us!

You can subscribe and unsubscribe from the context menu when you click on the kebab button on right side. If you're not able to do that, then would you mind share more details about the steps you followed so that i help you and It would be great if you share with us in which platform (Android or iOS) and which version you're facing this issue? 

Thanks!

**Reply by esahmed21** [reply, marked as answer]
@rywz Thank you for providing a recording and more information! That definitely a bug and should not happen, I'll create an issue internally for the team to fix it! 

Meanwhile would it be possible if you refresh using pull to refresh after unsubscribing, would that work for you?

Thanks!

**Reply by donmccurdy** [reply]
I'm encountering the same issue on iOS (iPhone and iPad). This is a pretty disruptive bug for me, as I can't subscribe to important issues and PRs without a desktop device.

**Reply by istudyatuni** [reply]
I found a workaround - if you tap faster on 3 dots, around 2-3 times, you can subscribe

**Reply by gokulyc** [reply]
When can we expect the fix?

**Reply by Zxjkasd** [reply]
اع

**Comment by kuraga** [comment]
Facing the same issue... See `Unsubscribe` on each of the issues/PRs...
GitHub Mobile v1.161.0 (786), Android 14.
@esahmed21 , what's the status?
Thanks!

**Reply by kuraga** [reply]
Moved to #123599 .
